Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -2040,5 +2040,5 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <string name="fab_scanner_tip_title">QR-Code-Scanner</string>
     <!-- XTXT: UQS Tooltip subtitle -->
     <string name="fab_scanner_tip_subtitle">Scannen Sie Tests, Zertifikate und Check-ins mit diesem Scanner.</string>
-    <string name="home_eol_message">Vielen Dank für die Nutzung der Corona-Warn-App. Während der Pandemie wurden durch die Nutzenden der Corona-Warn-App rund 9 Millionen positive Testergebnisse geteilt. Durch diese Teilung positiver Testergebnisse konnten Personen schnell vor einem erhöhten Infektionsrisiko gewarnt werden, das ihnen sonst unerkannt geblieben wäre. Viele Gewarnte wurden anschließend positiv getestet. Die Corona-Warn-App hat dank Ihrer Mithilfe ihren Zweck der frühzeitigen Warnung erfüllt und die Beendigung von Infektionsketten unterstützt.\n\nAb dem 1. Juni 2023 wird die Corona-Warn-App nicht mehr weiterentwickelt. Sie können weiterhin auf Ihre bereits in der App gespeicherten Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung.\n\nWenn Sie wichtige Warnmeldungen für unterschiedliche Gefahrenlagen erhalten wollen, empfehlen wir Ihnen die <a href="https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/NINA-Download/nina-download_node.html">Warn-App NINA</a> des Bundes.</string>
+    <string name="home_eol_message">Vielen Dank für die Nutzung der Corona-Warn-App. Während der Pandemie wurden durch die Nutzenden der Corona-Warn-App rund 9 Millionen positive Testergebnisse geteilt. Durch diese Teilung positiver Testergebnisse konnten Personen schnell vor einem erhöhten Infektionsrisiko gewarnt werden, das ihnen sonst unerkannt geblieben wäre. Viele Gewarnte wurden anschließend positiv getestet. Die Corona-Warn-App hat dank Ihrer Mithilfe ihren Zweck der frühzeitigen Warnung erfüllt und die Beendigung von Infektionsketten unterstützt.\n\nAb dem 1. Juni 2023 wird die Corona-Warn-App nicht mehr weiterentwickelt. Sie können weiterhin auf Ihre bereits in der App gespeicherten Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung.\n\nWenn Sie wichtige Warnmeldungen für unterschiedliche Gefahrenlagen erhalten wollen, empfehlen wir Ihnen die <a href="https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/NINA-Download/nina-download_node.html">Warn-App NINA des Bundesamts für Bevölkerungsschutz und Katastrophenhilfe (BBK)</a> (external link).</string>
```

Comment:
Why does this say "external link" in German? It should be "externer Link".

```suggestion
    <string name="home_eol_message">Vielen Dank für die Nutzung der Corona-Warn-App. Während der Pandemie wurden durch die Nutzenden der Corona-Warn-App rund 9 Millionen positive Testergebnisse geteilt. Durch diese Teilung positiver Testergebnisse konnten Personen schnell vor einem erhöhten Infektionsrisiko gewarnt werden, das ihnen sonst unerkannt geblieben wäre. Viele Gewarnte wurden anschließend positiv getestet. Die Corona-Warn-App hat dank Ihrer Mithilfe ihren Zweck der frühzeitigen Warnung erfüllt und die Beendigung von Infektionsketten unterstützt.\n\nAb dem 1. Juni 2023 wird die Corona-Warn-App nicht mehr weiterentwickelt. Sie können weiterhin auf Ihre bereits in der App gespeicherten Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung.\n\nWenn Sie wichtige Warnmeldungen für unterschiedliche Gefahrenlagen erhalten wollen, empfehlen wir Ihnen die <a href="https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/NINA-Download/nina-download_node.html">Warn-App NINA des Bundesamts für Bevölkerungsschutz und Katastrophenhilfe (BBK)</a> (externer Link).</string>
```

Reply:
@Ein-Tim, thanks for spotting it. iOS has been informed as well.

Reply:
Thank you!